common/jmeter/oidc/OIDC_Password_Grant_RBAC.jmx

```diff
@@ -0,0 +1,363 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.6.3">
```

Comment:
Use a common Jmeter version that was used for other scripts as well, don't just update versions. Eventually it won't then support the Jenkins build since there we have a fixed version.
jmeter="5.5"

Reply:
ack